src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/FileUtils.java

```diff
@@ -0,0 +1,56 @@
+package org.broadinstitute.hellbender.tools.spark.sv.utils;
+
+import htsjdk.samtools.*;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.gcs.BucketUtils;
+import org.broadinstitute.hellbender.utils.read.ReadUtils;
+
+import java.io.*;
+import java.util.Iterator;
+
+public final class FileUtils {
+
+    public static void writeLinesToSingleFile(final Iterator<String> linesToWrite, final String fileName) {
+        try ( final OutputStream writer =
+                      new BufferedOutputStream(BucketUtils.createFile(fileName)) ) {
+            while (linesToWrite.hasNext()) {
+                writer.write(linesToWrite.next().getBytes()); writer.write('\n');
+            }
+        } catch ( final IOException ioe ) {
+            throw new GATKException("Can't write "+fileName, ioe);
+        }
+    }
+
+    public static void writeSAMFile(final Iterator<SAMRecord> alignments, final SAMFileHeader header, final String outputName,
+                                    final boolean preOrdered) {
+        try (final SAMFileWriter writer = ReadUtils.createCommonSAMWriter(new File(outputName), null, header,
+                preOrdered, true, false) ) {
+            alignments.forEachRemaining(writer::addAlignment);
+        } catch ( final UncheckedIOException ie) {
+            throw new GATKException("Can't write SAM file to the specified location: " + outputName, ie);
+        }
+    }
+
+    public static boolean createDirInBucketToWriteTo(final String pathString) {
+        try {
+            Utils.nonNull(pathString);
+            if ( java.nio.file.Files.exists(java.nio.file.Paths.get(pathString)) )
```

Comment:
If possible, you should modify your imports so a fully qualified name isn't required for `java.io.file.` symbols.